lib/galaxy/util/__init__.py

```diff
@@ -58,6 +58,21 @@
 BINARY_CHARS = [ NULL_CHAR ]
 
 
+def remove_protocol_from_url( url ):
+    """ Supplied URL may be null, if not ensure http:// or https://
+    etc... is stripped off.
+    """
+    if url is None:
+        return url
+
+    # We have a URL
+    if url.find( '://' ) > 0:
```

Comment:
I prefer what is there I guess (refactored out of the toolshed), I makes fewer assumptions on the type of prefix that is being stripped off.